rs/sns/governance/src/topics.rs

```diff
@@ -232,14 +233,15 @@ impl Governance {
         };
 
         let Some(custom_proposal_topic_id) = custom_proposal_topic_id else {
-            return Ok(None);
+            // Fall back to legacy, action-specific proposal criticality (if a topic isn't defined).
+            return Ok((None, action.proposal_criticality()));
```

Comment:
Okay, I answered my question... so if this is the fallback, then maybe "proposal_criticality" on action should be renamed to be "default_proposal_criticality" and then you can add a note saying that proposal_criticality is determined primarily by topic, and that this fallback behavior is slated for removal?

Reply:
The downside of not making these notes or changing the name is that it looks like they're both weighted equally in a conceptual sense, and I think we should make it clear for future readers what the design intentions are.

Reply:
Great suggestion

I ended up removing `action.proposal_criticality` because it's confusing (only the case for custom proposals should be used from the legacy implementation, so it's not worth keeping it around).

While removing it, I found a place where wait-for-quite period was defined incorrectly (based on legacy criticality, rather than topic-based criticality) and fixed it.

Now we just have a concept of default criticality, and it's explained in the doc strings why it's Normal rather than Critical.